test/e2e/apps/deployment.go

```diff
@@ -746,6 +750,28 @@ func testIterativeDeployments(f *framework.Framework) {
 	Expect(framework.WaitForDeploymentWithCondition(c, ns, deploymentName, deploymentutil.NewRSAvailableReason, extensions.DeploymentProgressing)).NotTo(HaveOccurred())
 }
 
+func replicaSetHasDesiredReplicas(rsClient extensionsclient.ReplicaSetsGetter, replicaSet *extensions.ReplicaSet) wait.ConditionFunc {
```

Comment:
We can move this into test/e2e/framework/rs_util.go, and turn this into something like `WaitForReplicaSetDesiredReplicas`, and use the `Poll` and `pollShortTimeout` defined there.

Reply:
noted!